### PR TITLE
255 [Guides] Download Block

### LINKED ISSUE #255: [Guides] Download Block
Figma:
https://www.figma.com/design/0MPkm5mzMfI0UPS6DWdbKs/Optimus-%2F-Abcam-Website-Designs?node-id=3192-251&node-type=section&t=JMKqZiVW0sDgTdNk-0

Example page: https://www.abcam.com/en-us/technical-resources/guides/antibody-basics 
image

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #255 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.page/en-us/stories/films/maryann-martone
- After: https://255-guides-download-block--danaher-abcam--aemsites.hlx.page/en-us/technical-resources/guides/antibody-basics

### CODE REVIEW

File: component-filters.json [hunk 1]
```diff
@@ -70,7 +71,8 @@
       "button",
       "title",
       "abcam-button",
-      "media"
+      "media",
+      "download"
```
Comment (rgravitvl) [inline component-filters.json:75]: do we really need download at column??

File: component-models.json [hunk 2]
```diff
@@ -1673,5 +1673,62 @@
         "value": ""
       }
     ]
-  }  
+  },
+  {
+    "id": "download",
+    "fields": [
+      {
```
Comment (rgravitvl) [inline component-models.json:1680]: try to reuse the button fields

File: eds/blocks/download/download.js [hunk 3]
```diff
@@ -5,15 +5,18 @@ import { getMetadata } from '../../scripts/aem.js';
...
 export default function decorate(block) {
   const title = getMetadata('og:title');
-  const { href } = window.location;
-  const downloadBlock = div(
-    { class: 'flex flex-col bg-[#f2f2f2] rounded-lg p-6 max-w-[544px] mb-10' },
+  const downloadLink = block.querySelector('p > a');
+  const linkText = downloadLink.textContent;
+  const linkHref = downloadLink.href;
+  const downloadDiv = div(
+    { class: 'flex flex-col bg-[#f2f2f2] rounded-lg p-6 max-w-[544px] mt-6 mb-2' },
```
Comment (rgravitvl) [inline eds/blocks/download/download.js:12]: Can we get the colors from tailwind configs??

**Review by rgravitvl — DISMISSED** [review]

File: component-models.json (hunk 2 above)
Comment (sharanagoudapatil) [reply component-models.json]: I've updated the json.

File: component-filters.json (hunk 1 above)
Comment (sharanagoudapatil) [reply component-filters.json]: It's removed as of now not required.

File: eds/blocks/download/download.js (hunk 3 above)
Comment (sharanagoudapatil) [reply eds/blocks/download/download.js]: Will update once we get the color code from Sunny (UX Developer).

**Review by rgravitvl — APPROVED** [review]
Looks good to me

**Review by rgravitvl — APPROVED** [review]

**Review by pardeepgera23 — APPROVED** [review]

**Pull request merged**